doc/source/release/1.5.0-notes.rst

```diff
@@ -12,11 +12,11 @@ Python 3 compatibility
 This is the first NumPy release which is compatible with Python 3. Support for
 Python 3 and Python 2 is done from a single code base. Extensive notes on
 changes can be found at
-`<http://projects.scipy.org/numpy/browser/trunk/doc/Py3K.txt>`_.
+`<https://web.archive.org/web/20100814160313/http://projects.scipy.org/numpy/browser/trunk/doc/Py3K.txt>`_.
```

Comment:
It turns out this doc is actually included in the `numpy/doc/Py3K.rst.txt`, so the external archive link should be replaced with a reference to the internal doc.

Reply:
For now this is fine I think. That file needs to be moved into the actual docs, rather than being some random text file. And we don't want to have to go back and fix these release notes yet again when doing that (it's likely we forget).